.github/workflows/ci_tests.yaml

```diff
@@ -31,14 +31,21 @@ jobs:
           python-version: 3.8
 
       - name: Install packages
-        run: pip install black blackdoc flake8 pylint
+        run: |
+          pip install black blackdoc flake8 pylint
+          sudo apt-get install dos2unix
```

Comment:
Didn't know apt- get works on Windows?!

Reply:
This is style check and it runs on Ubuntu only.

Reply:
Ah right, all good then.